lib/scarpe/widgets.rb

```diff
@@ -19,3 +19,5 @@
 
 require_relative "text_widget"
 require_relative "link"
+
+require_relative "../../logger/logger.rb"
```

Comment:
This may want to be a require from lib/scarpe.rb. I don't think we want logging to be specific to widgets.